experimental/dds/attributable-map/src/localValues.ts

```diff
@@ -134,9 +128,7 @@ export class LocalValueMaker {
 			serializable.value = handle;
 		}
 
-		const translatedValue: unknown = parseHandles(serializable.value, this.serializer);
```

Comment:
I assume you need to do the same back-compat affordance here that you did in the other LocalValueMaker?

Reply:
Is this still coming?